### ISSUE TITLE
Update text related to uploaded files

### ISSUE THREAD

**Issue by techsmyth** [alkem-io/client-web#5098]
## Description

As a user the current text is scary

## Acceptance criteria

- [ ] Adjust the text under references: Click here for the Terms & Conditions

## Additional Context
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK1dPQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--3b168e5ca3e7e003ed67acc4073fc929c7922659/image.png)

**Comment by SimoneZaza** [comment]
@techsmyth is the content of the text still valid? Then I can try to rephrase it to make it sound less scary :)

**Comment by techsmyth** [comment]
@SimoneZaza the warning actually needs to be different. It is no longer that anyone with the link can see the files. It is that anyone who is able to access this entity can see the content. So I would suggest the message becomes something like: "Uploaded files are accessible to users who can access this entity. Click here for t&cs". It is still true that only Alkemio can delete the files for now - but not sure we want to scare users with that message here?

**Comment by SimoneZaza** [comment]
@techsmyth let's discuss this in the product meeting

**Comment by SimoneZaza** [comment]
@techsmyth first part seems a bit redundant: I think it's clear that the file is accessible to the people who can see the page where you're placing it... So I've just removed the first part.
